lib/yt/auth.rb

```diff
@@ -27,8 +29,8 @@ def url
 
     # @return [String] the email of an authenticated Google account.
     def email
-      response = HTTPRequest.new(email_params).run
-      response.body['email']
+      decoded_token = JWT.decode tokens['id_token'], nil, false
+      decoded_token[0]['email']
```

Comment:
JWT.decode method returns an array `[{...'email'=>'user@example.net'...}, {'alg'=>'RS256', ...}]` so I used the first element hash https://github.com/jwt/ruby-jwt#algorithms-and-usage